pp/integrations/plugin-packs/getting-started/how-to-guides/telegraf.md

```diff
@@ -10,7 +10,7 @@ import TabItem from '@theme/TabItem';
 The Telegraf agent collects metrics and computes statuses on the servers it monitors, and sends them to Centreon.
 Centreon's OpenTelemetry processor allows it to understand data sent in OpenTelemetry format.
 
-Developped by InfluxDB, the Telegraf agent can be installed on Windows servers. Centreon Engine is able to send the configuration of monitored resources to the agent (existing hosts, thresholds...), so that the agent can execute checks and compute the statuses of these resources.
+Developed by InfluxDB, the Telegraf agent can be installed on Windows servers. Centreon Engine is able to send the configuration of monitored resources to the agent (existing hosts, thresholds...), so that the agent can execute checks and compute the statuses of these resources.
```

Comment:
```suggestion
Developed by InfluxDB, the Telegraf agent can be installed on Windows or Linux servers. Centreon Engine is able to send the configuration of monitored resources to the agent (existing hosts, thresholds...), so that the agent can execute checks and compute the statuses of these resources.
```